### PR TITLE
Set the extended load test override defaults to true

### PR DESCRIPTION
**Pull request by maniSbindra**
This PR is to resolve issue #1036 . As described in #1036 to resolve the issue changes also need to be made in k8s.io/test-infra, for which PR https://github.com/kubernetes/test-infra/pull/16818 has been created

### CODE REVIEW

File: clusterloader2/testing/load/experimental/overrides/enable_configmaps.yaml [hunk 1]
```diff
@@ -1 +0,0 @@
-ENABLE_CONFIGMAPS: true
```
Comment (mm4tt) [inline clusterloader2/testing/load/experimental/overrides/enable_configmaps.yaml]: We cannot delete these files yet as they are being listed in job definitions. If we delete them the test jobs will start failing
We need 3 separate PRs, in the following order
1. Change the defaults to true
2. Change job definitions to not use override files (in test-infra repo)
3. Delete override files

Comment (maniSbindra) [reply clusterloader2/testing/load/experimental/overrides/enable_configmaps.yaml]: @mm4tt  I have reverted the file deletions. Should I delete the test-infra repo PR, or can I just I put a comment in the test-infra repo PR that that can only be merged after this PR is merged? Thanks Mani.

File: clusterloader2/testing/load/config.yaml [hunk 2]
```diff
@@ -17,13 +17,13 @@
 {{$ENABLE_CHAOSMONKEY := DefaultParam .ENABLE_CHAOSMONKEY false}}
 {{$PROMETHEUS_SCRAPE_KUBE_PROXY := DefaultParam .PROMETHEUS_SCRAPE_KUBE_PROXY true}}
 {{$ENABLE_PROMETHEUS_API_RESPONSIVENESS := DefaultParam .ENABLE_PROMETHEUS_API_RESPONSIVENESS false}}
-{{$ENABLE_CONFIGMAPS := DefaultParam .ENABLE_CONFIGMAPS false}}
-{{$ENABLE_DAEMONSETS := DefaultParam .ENABLE_DAEMONSETS false}}
-{{$ENABLE_JOBS := DefaultParam .ENABLE_JOBS false}}
-{{$ENABLE_PVS := DefaultParam .ENABLE_PVS false}}
-{{$ENABLE_SECRETS := DefaultParam .ENABLE_SECRETS false}}
-{{$ENABLE_STATEFULSETS := DefaultParam .ENABLE_STATEFULSETS false}}
-{{$ENABLE_NETWORKPOLICIES := DefaultParam .ENABLE_NETWORKPOLICIES false}}
```
Comment (mm4tt) [inline clusterloader2/testing/load/config.yaml]: Please don't touch networkpolicies, I think they are not ready yet.
Sorry, it should've been mentioned in the issue

File: clusterloader2/testing/load/experimental/overrides/enable_configmaps.yaml (hunk 1 above)
Comment (maniSbindra) [reply clusterloader2/testing/load/experimental/overrides/enable_configmaps.yaml]: I have closed the  test-infra PR.

File: clusterloader2/testing/load/config.yaml [hunk 3]
```diff
@@ -17,13 +17,13 @@
 {{$ENABLE_CHAOSMONKEY := DefaultParam .ENABLE_CHAOSMONKEY false}}
 {{$PROMETHEUS_SCRAPE_KUBE_PROXY := DefaultParam .PROMETHEUS_SCRAPE_KUBE_PROXY true}}
 {{$ENABLE_PROMETHEUS_API_RESPONSIVENESS := DefaultParam .ENABLE_PROMETHEUS_API_RESPONSIVENESS false}}
-{{$ENABLE_CONFIGMAPS := DefaultParam .ENABLE_CONFIGMAPS false}}
-{{$ENABLE_DAEMONSETS := DefaultParam .ENABLE_DAEMONSETS false}}
-{{$ENABLE_JOBS := DefaultParam .ENABLE_JOBS false}}
-{{$ENABLE_PVS := DefaultParam .ENABLE_PVS false}}
```
Comment (mm4tt) [inline clusterloader2/testing/load/config.yaml]: I think we disable PVs in kubemark tests, see https://github.com/kubernetes/perf-tests/issues/803
Because of that, we need to introduce a way to disable it in kubemark tests once we switch the default to true.
Sorry, this should've been also mentioned in the issue.

I think the easiest way to tackle this would be:
1. Sent a PR that will rename these variables to have CL2_ prefix, e.g. replace ENABLE_PVS to CL2_ENABLE_PVS. If a variable begins with CL2 it's possible to change it value by overriding env. Changing the name should be a no-op change because the name is not referenced anywhere outside this repo
2. Sent a PR to test-infra and change this file - https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml by adding the following arg to all kubemark jobs (name contains kubemark) 
   ```
   - --env=CL2_ENABLE_PVS=false # TODO(https://github.com/kubernetes/perf-tests/issues/803): Fix me
   ```
3. Once you do it you should be good to go to change the default to false

File: clusterloader2/testing/load/config.yaml (hunk 2 above)
Comment (maniSbindra) [reply clusterloader2/testing/load/config.yaml]: reverted network policy changes. Default set to false again

**Comment by mm4tt** [comment]
Thanks @maniSbindra!
This is really great, waiting for the PR that deals with ENABLE_PVS before approving this one.

**Comment by maniSbindra** [comment]
Thanks @mm4tt  will work on that change a bit later today

**Comment by maniSbindra** [comment]
@mm4tt I have created the PR https://github.com/kubernetes/perf-tests/pull/1126

File: clusterloader2/testing/load/config.yaml (hunk 3 above)
Comment (mm4tt) [reply clusterloader2/testing/load/config.yaml]: Nice, point 1 is done. 
Now we need to do point 2 and we'll be good to submit this PR

File: clusterloader2/testing/load/config.yaml [hunk 4]
```diff
@@ -17,12 +17,12 @@
 {{$ENABLE_CHAOSMONKEY := DefaultParam .ENABLE_CHAOSMONKEY false}}
 {{$PROMETHEUS_SCRAPE_KUBE_PROXY := DefaultParam .PROMETHEUS_SCRAPE_KUBE_PROXY true}}
 {{$ENABLE_PROMETHEUS_API_RESPONSIVENESS := DefaultParam .ENABLE_PROMETHEUS_API_RESPONSIVENESS false}}
-{{$ENABLE_CONFIGMAPS := DefaultParam .CL2_ENABLE_CONFIGMAPS false}}
-{{$ENABLE_DAEMONSETS := DefaultParam .CL2_ENABLE_DAEMONSETS false}}
-{{$ENABLE_JOBS := DefaultParam .CL2_ENABLE_JOBS false}}
+{{$ENABLE_CONFIGMAPS := DefaultParam .CL2_ENABLE_CONFIGMAPS true}}
+{{$ENABLE_DAEMONSETS := DefaultParam .CL2_ENABLE_DAEMONSETS true}}
+{{$ENABLE_JOBS := DefaultParam .CL2_ENABLE_JOBS true}}
 {{$ENABLE_PVS := DefaultParam .CL2_ENABLE_PVS false}}
```
Comment (mm4tt) [inline clusterloader2/testing/load/config.yaml]: Let' disable it back and send another PR in test-infra to change the kubemark presubmit jobs, we forgot about them. It's this file - https://github.com/kubernetes/test-infra/blob/6cb0718bf7f94883199ea290284155b29523db98/config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml#L417

**Comment by mm4tt** [comment]
Sorry, @maniSbindra 

I got swamped with other work. Let me see why kubemark test is failing

**Comment by mm4tt** [comment]
The https://github.com/kubernetes/perf-tests/pull/1145 should fix the failing kubemark tests once it gets merged.
